src/deployment/flavors.md

```diff
@@ -29,5 +32,6 @@ The following packages are listed alphabetically:
 [Flutter Ready to Go]: {{site.medium}}/flutter-community/flutter-ready-to-go-e59873f9d7de
 [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor]: {{site.medium}}/@animeshjain/build-flavors-in-flutter-android-and-ios-with-different-firebase-projects-per-flavor-27c5c5dac10b
 [Flutter 1.17 — no more Flavors, no more iOS Schemas. Command argument that changes everything]: {{site.medium}}/@tatsu.ukraine/flutter-1-17-no-more-flavors-no-more-ios-schemas-command-argument-that-solves-everything-8b145ed4285d
+[Multi-environment Flutter Projects with Flavors]: {{https://sebastien-arbogast.com/2022/05/02/multi-environment-flutter-projects-with-flavors/
```

Comment:
I believe the `{{` at the start of the URL is incorrect. Please remove. 

For context, we use `{{variable.name}}` to interpolate site wide variables in the publishing engine.

Reply:
Thanks. I fixed it.